miasm2/jitter/llvmconvert.py

```diff
@@ -69,6 +70,22 @@ def __init__(self, name="mod"):
         self.target_machine = target.create_target_machine()
         self.init_exec_engine()
 
+    def canonize_label_name(self, label):
+        """Canonize @label names to a common form.
+        @label: str or asmlabel instance"""
+        if isinstance(label, str):
+            return label
+        if not isinstance(label, LocKey):
+            raise ValueError("label must either be str or LocKey")
+
+        name = self.ir_arch.symbol_pool.loc_key_to_name(label)
+        offset = self.ir_arch.symbol_pool.loc_key_to_offset(label)
+
+        if offset is None:
+            return "%s" % name
```

Comment:
name only used if offset is None, so `name = ` can be moved here

Reply:
done